### DISCUSSION TITLE
How to 'select' the currently playing track

### DISCUSSION THREAD

**Discussion by seamusdemora** [cmus/cmus#1392] [General]
I mostly use the "Play Queue" (4) to listen to new music. I select an artist (or album) from (1), and just hit the `e` key to add the songs/tracks to the Play Queue. As I'm listening, I hear something I like, and I want to add it to a play list. Currently, I can only do this by changing from (4) to (1), finding the song there, and then adding to the playlist with the `y` key. 

Is there not a quicker way? IOW, is there some way to "Select" the currently-playing track in (4), and use the `y` key??

**Comment by gavtroy** [comment]
I don't think you can do it natively, but you can bind a short script like:
```
:bind common W shell cmus-remote -P "$(cmus-remote -C 'format_print %f')"; cmus-remote -C "echo Added to playlist!"
```
and then hitting W should do what you want.

**Reply by seamusdemora** [reply]
That did it!  Many thanks!
